core/services/ardupilot_manager/setup.py

```diff
@@ -13,5 +13,12 @@
     long_description_content_type="text/markdown",
     packages=setuptools.find_packages(),
     python_requires=">=3.6",
-    install_requires=["appdirs==1.4.4", "packaging==20.4", "smbus2==0.3.0"],
+    install_requires=[
+        "appdirs==1.4.4",
+        "packaging==20.4",
+        "smbus2==0.3.0",
+        "fastapi==0.63.0",
+        "uvicorn==0.13.4",
+        "python-multipart==0.0.5",
```

Comment:
where is this being used ?

Reply:
This is for the firmware-upload part. I mistakenly committed it already.